### PR TITLE
doc: fix jail, add missed-too-many-blocks

### PR DESCRIPTION
**Pull request by ggutoski**
Is everything correct?  Is there something more I should add?

### CODE REVIEW

File: documentation/docs/validator-zone/troubleshoot/unjail.md [hunk 1]
```diff
@@ -6,21 +6,13 @@ slug: /validator-zone/troubleshoot/unjail
...
-tofnd check: passed
-broadcaster check: passed
-operator check: failed (health check to operator MY_VALIDATOR_ADDRESS failed due to the following issues: {"missed_too_many_blocks":true})
-```
-
-You can also see your 'jailed' status via the Cosmos command `axelard q staking validators`.
+"Jail" is a Tendermint/Cosmos concept---it is not specific to Axelar.  If your validator misses 50 or more of the last 100 blocks then your tendermint status becomes `jailed`.  You can see your 'jailed' status via the Cosmos command `axelard q staking validators`.
 
 :::tip
 Axelar-core currently uses the word "jail" to describe a different Axelar-specific status in the context of eligibility to participate in keygen/sign protocols.  This terminology can be confusing and we intend to change it in future versions of axelar-core.  To see whether your validator's has this Axelar-specific jail status use `axelard q snapshot validators`.
```
Comment (haiyizxx) [inline documentation/docs/validator-zone/troubleshoot/unjail.md:12]: is it the ` tss_illegibility_info`? I think we do not re-use jail

Comment (milapsheth) [reply documentation/docs/validator-zone/troubleshoot/unjail.md:12]: Yeah, maybe we should switch to using that

**Review by milapsheth — APPROVED** [review]

File: documentation/docs/validator-zone/troubleshoot/unjail.md (hunk 1 above)
Comment (ggutoski) [reply documentation/docs/validator-zone/troubleshoot/unjail.md:12]: I believe we do re-use "jail".  The following commands produce different sets of validators with `jailed: true`:
```
axelard q staking validators | grep "jailed: true"
axelard q snapshot validators | grep "jailed: true"
```

Comment (milapsheth) [reply documentation/docs/validator-zone/troubleshoot/unjail.md:12]: The snapshot only lists validators that were included in the latest TSS snapshot, not all validators (i.e the ones that are unbonded/inactive). `staking` includes all validators (until they have been de registered). So, jailed snapshot validators should be a subset of jailed staking validators.

**Pull request merged**